python/cuml/benchmark/run_benchmarks.py

```diff
@@ -82,6 +83,9 @@ def extract_param_overrides(params_to_sweep):
           python run_benchmarks.py --dataset higgs --default-size \
                 RandomForestClassifier LogisticRegression
 
+          # Print available algorithms and exit
+          python run_benchmarks.py --algos
```

Comment:
No need to list this here, it will be automagically generated by argparse since you used the `help=` option.